tests/wpunit/DocumentGarbageCollectionTest.php

```diff
@@ -55,7 +55,7 @@ public function testQueriesAreDeletedByJob() {
 					[
 						'post_type' => 'graphql_document',
 						'post_date' => $date_string,
-						'post_content' => sprintf( "query Saved_%d { typename }", $counter ),
+						'post_content' => sprintf( "query Saved_%d { __typename }", $counter ),
```

Comment:
Doing the rule validation, brought to attention some invalid test query strings.  :)